automation_tools/__init__.py

```diff
@@ -1088,6 +1088,119 @@ def downstream_install(admin_password=None, run_katello_installer=True):
     else:
         return installer_options
 
+def repofile_install(admin_password=None, run_katello_installer=True, repo_url=None):
+    """Task to install Satellite 6.3 via repo files
+
+    The following environment variables affect this command:
+
+    ADMIN_PASSWORD
+        Optional, defaults to 'changeme'. Foreman admin password.
+    REPO_FILE_URL
+        URL for the compose repository file to fetch.
+    SATELLITE_VERSION
+        Satellite version.
+
+    """
+    # For Satellite 6.1 and 6.2 call downstream_install only.
+    if os.environ.get('SATELLITE_VERSION') != '6.3':
```

Comment:
These new tasks are not atomic, the flow is controlled by `product_install` which decides what tasks to call depending of the type of installation being done. That said I think is responsibility of `product_install` to check if `downstream_install` or some other task is called depending on the satellite version.

This apply for the other task added.